std/datetime.d

```diff
@@ -2433,28 +2434,61 @@ public:
 
 
     /++
-        Returns a $(D time_t) which represents the same time as this
-        $(LREF SysTime).
+        Converts this $(LREF SysTime) to unix time (i.e. seconds from midnight,
+        January 1st, 1970 in UTC).
 
-        Note that like all conversions in std.datetime, this is a truncating
-        conversion.
+        The C standard does not specify the representation of time_t, so it is
+        implementation defined. On POSIX systems, unix time is equivalent to
+        time_t, but that's not necessarily true on other systems (e.g. it is
+        not true for the Digital Mars C runtime). So, be careful when using unix
+        time with C functions on non-POSIX systems.
+
+        By default, the return type is time_t (which is normally an alias for
+        int on 32-bit systems and long on 64-bit systems), but if a different
+        size is required than either int or long can be passed as a template
+        argument to get the desired size.
+
+        If the return type is int, and the result can't fit in an int, then the
+        closest value that can be held in 32 bits will be used (so $(D int.max)
+        if it goes over and $(D int.min) if it goes under). However, no attempt
+        is made to deal with integer overflow if the return type is long.
 
-        If $(D time_t) is 32 bits, rather than 64, and the result can't fit in a
-        32-bit value, then the closest value that can be held in 32 bits will be
-        used (so $(D time_t.max) if it goes over and $(D time_t.min) if it goes
-        under).
+        Params:
+            T = The return type (int or long). It defaults to time_t, which is
+                normall 32 bits on a 32-bit system and 64 bits on a 64-bit
+                system.
+
+        Returns:
+            A signed integer representing the unix time which is equivalent to
+            this SysTime.
```

Comment:
Minor nit, not necessary for pulling. But grammatically, it reads that "unix time is equivalent to this SysTime", it reads very oddly.

Better: "A signed integer representing the equivalent unix time for this SysTime"